infra/modules/kubevela-addons/definitions.tf

```diff
@@ -162,6 +162,33 @@ resource "kubectl_manifest" "componentdefinition_bucket" {
   })
 }
 
+resource "kubectl_manifest" "traitdefinition_cardano_node_connector" {
+  force_new = true
```

Comment:
Out of curiosity, why is this needed? Do `TraitDefinitions` not support in-place modification?

Reply:
you are right @shlevy 

removed :)